lib/phx_tools_web/live/phx_tools_components.ex

```diff
@@ -36,42 +36,76 @@ defmodule PhxToolsWeb.PhxToolsComponents do
   @spec os_landing_card(assigns()) :: rendered()
   def os_landing_card(%{live_action: :index} = assigns) do
```

Comment:
Can you refactor this function because many of the elements are the same between supported and unsupported OS?